consensus/module.go

```diff
@@ -77,34 +77,42 @@ type consensusModule struct {
 type ConsensusDebugModule interface {
 	SetHeight(uint64)
 	SetRound(uint64)
-	SetStep(typesCons.HotstuffStep)
+	//REFACTOR: This should accept typesCons.HotstuffStep.
```

Comment:
NIT: here and elsewhere. Please add spaces after a `//`
```suggestion
	// REFACTOR: This should accept typesCons.HotstuffStep.
```